pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools>=34.4", "wheel"]
+build-backend = "setuptools.build_meta:__legacy__"
+
+[tool.black]
+skip-string-normalization = true
```

Comment:
I'm a little surprised to see black configuration in here, though I like the idea of using it.

Reply:
That's just an incidental change. I'll remove it as it's irrelevant to this PR.